### ISSUE TITLE
TypeError: 'Config' object is not callable

### ISSUE THREAD

**Issue by matanster** [chmp/ipytest#11]
```
import ipytest
ipytest.config(rewrite_asserts=True, magics=True)

```
> TypeError: 'Config' object is not callable

Jupyter installed with Anaconda, versions:

> Name                    Version                   Build  Channel
> jupyter                   1.0.0                    py36_4  
> jupyter_client            5.2.3                    py36_0  
> jupyter_console           5.2.0            py36he59e554_1  
> jupyter_contrib_core      0.3.3                      py_2    conda-forge
> jupyter_contrib_nbextensions 0.5.0                    py36_0    conda-forge
> jupyter_core              4.4.0            py36h7c827e3_0  
> jupyter_highlight_selected_word 0.2.0                    py36_0    conda-forge
> jupyter_latex_envs        1.4.4                    py36_0    conda-forge
> jupyter_nbextensions_configurator 0.4.0                    py36_0    conda-forge
> jupyterlab                0.32.1                   py36_0  
> jupyterlab_launcher       0.10.5                   py36_0  
> 

Any advice?

**Comment by babaMar** [comment]
Try with `ipytest.config.rewrite_asserts = True`

**Comment by matanster** [comment]
Thanks a lot. It works, and so does:

`import ipytest; ipytest.config.rewrite_asserts=True; ipytest.config.magics=True`

Thanks so much for this library.

**Comment by babaMar** [comment]
Thanks to @chmp! Guess you can close the issue

**Comment by matanster** [comment]
Unless I'm missing much though, the readme should be updated first, as it
currently suggests the opposite of this solution as the suggested setup.

**Comment by chmp** [comment]
@matanster the readme tracks the development version and not the latest release. I agree that it is confusing at the moment and should be fixed. The code in the readme will work with `pip install 'ipytest==0.5.0b3'`. 

The currently released version used the following setup:

```python
import ipytest
import ipytest.magics

ipytest.config.rewrite_asserts = True
```

**Comment by matanster** [comment]
@chmp thanks for the note, this makes sense. maybe leave this issue open so people find it in search

**Comment by matanster** [comment]
does `__file__` really also have to be set manually or via any of the notebook tricks for getting the notebook's own file name? is that also the case in the development version or you've adopted anything from e.g. [here](https://stackoverflow.com/questions/12544056/how-do-i-get-the-current-ipython-notebook-name)?

**Comment by chmp** [comment]
@matanster I cleaned everything up and moved development into its own branch (`develop`), as it should have been from the beginning. The current `master` should reflect how to use `ipytest` when installed from pip. Thanks again for bringing up this issue.

With regards to setting `___file__`: it still has to be done manually. I added it also the readme. At the moment there is no way to get the filename from inside the kernel. All the examples you have provided are using the frontend in some way. However, there is a very high chance this code will break, when you run the notebook for example in nbval. Therefore, I don't think it's a good idea to add such code to ipytest.
